ts/sagas/wallet.ts

```diff
@@ -151,6 +154,26 @@ const navigateTo = (routeName: string, params?: object) => {
   return NavigationActions.navigate({ routeName, params });
 };
 
+export type NodoErrors = detailEnum | "GENERIC_ERROR" | "MISSING_PAYMENT_ID";
+export type PaymentManagerErrors = "GENERIC_ERROR"; // no specific errors are available
+export type PagoPaErrors = NodoErrors | PaymentManagerErrors;
```

Comment:
mmh `NodoErrors` already includes `GENERIC_ERROR` that is also aliased to `PaymentManagerErrors`, it looks like `PagoPaErrors === NodoErrors` ?

Reply:
It is for the time being, since pagoPA's PM does not provide any meaningful error messages (or error codes, for that matter (e.g. when a payment cannot be made b/c it has already been made, or the payment Id is not valid, a 401 is returned)). 

So, as long as the a thorough documentation of the list of possible errors is not received from pagoPA, all PM errors are mapped to `GENERIC_ERROR`. The code, though, already "supports" possible future errors and, by changing `extractPaymentManagerError`'s implementation, the integration with the new errors should be straightforward.